index.html

```diff
@@ -0,0 +1,227 @@
+                    <li>2018</li>
+                </ul>
+                <div>
+                    <p class="uniqe-para">A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.</p>
+                </div>
+                <div class="nav-source">
+                    <ul>
+                        <li>html</li>
+                        <li>JavaScript</li>
+                        <li>css</li>
+                    </ul>
+                </div>
+                <div class="button-m">
+                    <button type="button">See Project</button>
+                </div>
+            </div>
+            <div class="back-image-file-3 all-image-pro mid-end"></div>
+        </article>
+    </section>
+    <section class="about-me" id="about-1">
+        <article>
+            <div class="about-me-page">
+                <h1 class="header-1">About Myself</h1>
+                <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <div class="let-s">
+                    <h1>LET'S CONNECT</h1>
+                    <div class="vector">
+                        <ul>
+                            <li><a href="https://github.com/girma336" class="twitter"></a></li>
+                            <li><a href="#" class="linkedin"></a></li>
+                            <li><a href="#" class="medium"></a></li>
+                            <li><a href="#" class="github"></a></li>
+                            <li><a href="#" class="hand"></a></li>
+                        </ul>
+                    </div>
+                </div>
+                <div class="button-m">
+                    <button type="button">See Project</button>
+                </div>
+            </div>
+            <div class="lists-of-frame">
+                <ul>
+                    <li class="language"><h1>Language</h1>
+                        <div class="up-down">
+                        <span  id="odd-span"  class="glyphicon glyphicon-chevron-right"></span></p>
+                        <span  id="new-span"  class="glyphicon glyphicon-chevron-down"></span></p>
+                        </div>
+                        
+                    
+                    <ul class="hovers">
+                        <li>
+                            <span class="list-1"></span>
+                             <p>JavaSript</p>
+                        </li>
+                        <li>  
+                            <span class="list-2"></span>                                
+                            <p>Htlm</p>
+                        </li>
+                        <li>
+                            <span class="list-3"></span>                             
+                            <p>Css</p>
+                        </li>
```

Comment:
### Your Hover Cards

![image](https://user-images.githubusercontent.com/93039359/194262327-34f0127f-5b90-40d6-84d8-3294ce715414.png)

- [ ] You can see that your hover cards are overlapping a grey line which is a border line of your Language heading. Kindly position your cards according to the Figma template :blush:

Reply:
Fixed,